### PR TITLE
Feat/add cloudweb in order menu

### PR DESCRIPTION
**Pull request by euhmeuh**
MFRWW-337

### CODE REVIEW

File: client/app/components/sidebar-menu/sidebar-menu.config.js [hunk 1]
```diff
@@ -6,16 +6,31 @@ angular.module("App").run(($q, SidebarMenu, Products, User, atInternet, constant
...
+            .then(([domainOrderUrl, cloudWebOrderUrl, emailproOrderUrl, office365OrderUrl]) => {
                 menuOptions.push({
                     id: "order-domain",
                     title: translator.tr("navigation_left_order_domain"),
                     icon: "ovh-font ovh-font-domain",
                     href: domainOrderUrl,
                     target: "_blank"
                 });
+                menuOptions.push({
+                    id: "order-hosting",
```
[the remark this answers is not in the document]
Comment (euhmeuh) [reply client/app/components/sidebar-menu/sidebar-menu.config.js]: Oopsie!

File: constants.config.js [hunk 2]
```diff
@@ -1727,6 +1727,24 @@ module.exports = {
                 TN : "https://www.ovh.com/tn/hebergement-web/",
                 WE : "http://www.ovh.com/us/"
             },
+            cloudweb_order : {
+                FR : "https://www.ovh.com/fr/order/domain/#/legacy/domain/search?options=~(hosting~'cloudweb1)",
```
Comment (FredericEspiau) [inline constants.config.js]: Didn't you already have the base URL somewhere where you could append `/order/domain/#/legacy/domain/search?options=~(hosting~'cloudweb1)` each time ?

Comment (euhmeuh) [reply constants.config.js]: That would have been wonderful.

Comment (FredericEspiau) [reply constants.config.js]: You're wonderful baby ❤️

Comment (euhmeuh) [reply constants.config.js]: :heart:

**Comment by euhmeuh** [comment]
Please squash before merging, there's a lot of useless commits

**Pull request merged**